README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Version 
 
-Current version: 1.3.2
+Current version: 1.4
```

Comment:
Should we be using a full 3 part version number, i.e. 1.4.0 ?

Reply:
Yes.